webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -801,90 +833,93 @@ public static float fitText(Font font, String text, Rectangle rect, float maxFon
         return potentialFontSize;
     }
 
-    private static LinkedList<Chunk> splitScrambleToLineChunks(String paddedScramble, Font scrambleFont, float scrambleColumnWidth) {
+    private static LinkedList<Chunk> splitTextToLineChunks(String text, Font scrambleFont, float scrambleColumnWidth) {
         float availableScrambleWidth = scrambleColumnWidth - 2*SCRAMBLE_PADDING_HORIZONTAL;
 
-        int startIndex = 0;
-        int endIndex = 0;
         LinkedList<Chunk> lineChunks = new LinkedList<Chunk>();
-        while(startIndex < paddedScramble.length()) {
-            // Walk forwards until we've grabbed the maximum number of characters
-            // that fit in a line, we've run out of characters, or we hit a newline.
-            float substringWidth;
-            for(endIndex++; endIndex <= paddedScramble.length(); endIndex++) {
-                if(paddedScramble.charAt(endIndex - 1) == '\n') {
-                    break;
-                }
-                String scrambleSubstring = NON_BREAKING_SPACE + paddedScramble.substring(startIndex, endIndex) + NON_BREAKING_SPACE;
-                substringWidth = scrambleFont.getBaseFont().getWidthPoint(scrambleSubstring, scrambleFont.getSize());
-                if(substringWidth > availableScrambleWidth) {
-                    break;
-                }
-            }
-            // endIndex is one past the best fit, so remove one character and it should fit!
-            endIndex--;
-
-            // If we're not at the end of the scramble, make sure we're not cutting
-            // a turn in half by walking backwards until we're right before a turn.
-            // Any spaces added for padding after a turn are considered part of
-            // that turn because they're actually NON_BREAKING_SPACE, not a ' '.
-            int perfectFitEndIndex = endIndex;
-            if(endIndex < paddedScramble.length()) {
-                while(true) {
-                    if(endIndex < startIndex) {
-                        // We walked all the way to the beginning of the line
-                        // without finding a good breaking point. Give up and break
-                        // in the middle of a turn =(.
-                        endIndex = perfectFitEndIndex;
+        String[] lineList = text.split("\n");
+        
+        for (String line:lineList){
```

Comment:
spacing nit: `){` -> `) {`

Reply:
I did not find it. Are you looking at the removed line?

Reply:
No, I'm looking at line 834 here:

![image](https://user-images.githubusercontent.com/277474/33233982-ecbc1bb6-d1d3-11e7-9d67-d6d258766823.png)

Reply:
Fixed